src/coreclr/jit/lower.cpp

```diff
@@ -5889,20 +5889,19 @@ GenTree* Lowering::CreateFrameLinkUpdate(FrameLinkAction action)
 //  64-bit  32-bit                                    CORINFO_EE_INFO
 //  offset  offset  field name                        offset                  when set
 //  -----------------------------------------------------------------------------------------
-//  +00h    +00h    GS cookie                         offsetOfGSCookie
```

Comment:
`CORINFO_EE_INFO::offsetOfGSCookie` and `CORINFO_EE_INFO::offsetOfFrameVptr` should be deleted

(Also, bump JIT/EE interface GUID.)